.github/workflows/dry-run-publish.yaml

```diff
@@ -0,0 +1,23 @@
+name: Dry Run Publish
+on:
+  pull_request:
+    branches:
+      - changeset-release/main
+      - changeset-release/release/v*
+
```

Comment:
@alionazherdetska can you add [workflow permissions](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token) to this workflow (see GitHub alert). Read permissions should be enough to run this task.